doc/releases/release-notes-3.2.rst

```diff
@@ -364,9 +428,57 @@ Libraries / Subsystems
   * Added :c:macro:`LOG_RAW` for logging strings without additional formatting.
     It is similar to :c:macro:`LOG_PRINTK` but do not append ``<cr>`` when new line is found.
 
+* LoRaWAN
+
+* Modbus
+
+* Power management
+
+* RTIO
+
+* SD Subsystem
+
+* Settings
+
+* Shell
+
+* Storage
+
+* Testsuite
+
+* Tracing
+
 HALs
 ****
 
+* Atmel
+
+* Espressif
+
+* GigaDevice
+
+* Nordic
+
+* NXP
+
+* Renesas
+
+* RPi Pico
+
+* Silabs
+
+* ST
+
+* STM32
+
+* Wurth Elektronik
+
+* Telink
+
+* TI
+
+* Xtensa
```

Comment:
should this be alphabetically sorted?

Reply:
Mostly... was merging a couple lists for this and I may have missed something to avoid shuffling existing entries thinking it may make it easier to solve merges if someone has this started already.

Reply:
Should be more reasonable now.